ui/admin/app/components/form/app-token/read/index.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+
+export default class FormAppTokenReadComponent extends Component {
+  @service intl;
+
+  statusConfig = {
+    active: { color: 'success' },
+    expired: { color: 'critical' },
+    revoked: { color: 'critical' },
+    stale: { color: 'critical' },
+    unknown: { color: 'neutral' },
+  };
+
+  /**
+   * Returns status badge configuration for app tokens
+   * @returns {object}
+   */
+  get statusBadge() {
+    const status = this.args.model?.status;
+    if (!status) return { text: '', color: 'neutral' };
+
+    const config = this.statusConfig[status] || { color: 'neutral' };
+    return {
+      text: this.intl.t(`resources.app-token.status.${status}`),
+      color: config.color,
+    };
+  }
+
+  /**
+   * Returns scope information (icon, text) based on scope type
+   * @returns {object}
+   */
+  get scopeInfo() {
+    const scope = this.args.model.scope;
+
+    if (scope.isGlobal) {
+      return {
+        icon: 'globe',
+        text: this.intl.t('resources.scope.types.global'),
+      };
+    }
+
+    if (scope.isOrg) {
+      return {
+        icon: 'org',
+        text: this.intl.t('resources.scope.types.org'),
+      };
+    }
+
+    return {
+      icon: 'grid',
+      text: this.intl.t('resources.scope.types.project'),
```

Comment:
I see we are using `text` for the display. This is showing up as `Global`, `Org`, or `Project` depending on the scope level. We should be using the `scope.displayName` here to show the scope name or id if it has no name.